iac-extensions/arm/src/test/resources/checks/UnusedVariablesCheckTest/containsUnresolvableVariable.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Is is possible to provide an equivalent for Bicep (with an unresolvable variable)?

Reply:
No, it can only happen in JSON